.github/workflows/ci2.yml

```diff
@@ -137,6 +137,62 @@ jobs:
                             --build-arg DAOS_JAVA_BUILD=no
                             --build-arg DAOS_BUILD_TYPE=dev
                             --build-arg COMPILER
+  BuildARM:
+    name: Build DAOS (ARM64)
+    runs-on: [self-hosted, ARM64]
```

Comment:
Do we want to do this on every PR or just landings?

Reply:
Let's start with landings

Reply:
Then you need to move these changes to https://github.com/daos-stack/daos/blob/master/.github/workflows/landing-builds.yml

Then it will trigger for landing builds, but also for PRs that make changes to the landing build so so this PR to add it will still be tested.